client/src/test/java/com/aws/greengrass/cli/commands/DeploymentCommandTest.java

```diff
@@ -195,6 +196,22 @@ void GIVEN_WHEN_failure_handling_policy_provided_with_invalid_words_THEN_invalid
         assertThat(exitCode, is(2));
     }
 
+    @Test
+    void GIVEN_deployment_cancel_When_deploymentId_provided_THEN_request_sent() {
+        int exitCode = runCommandLine("deployment", "cancel", "-i", "uuid");
+        CancelLocalDeploymentRequest request = new CancelLocalDeploymentRequest();
+        request.setDeploymentId("uuid");
+        verify(nucleusAdapteripc).cancelLocalDeployment(request);
+        assertThat(exitCode, is(0));
+    }
+
+    @Test
+    void GIVEN_deployment_cancel_When_deploymentId_not_provided_THEN_request_sent() {
+        int exitCode = runCommandLine("deployment", "cancel");
+        verify(nucleusAdapteripc, never()).cancelLocalDeployment(any());
+        assertThat(exitCode, is(2));
+    }
+
```

Comment:
Do we need a test like GIVEN_deployment_cancel_When_deploymentId_provided_isValid_THEN_xxxx

Reply:
is that the same as `GIVEN_deployment_cancel_When_deploymentId_provided_THEN_request_sent`? The actual deployment id validation is done on server side so we can't test anything on client side more than if an argument is supplied or not.